README.md

```diff
@@ -146,7 +146,8 @@ A server running WinRM must also be configured properly to allow outside connect
     winrm quickconfig -q
 
 The Chef and Ohai gem installations (that occur during bootstrap) take more
-memory than the default 150MB WinRM allocates per shell -- this can slow down
+memory than the default 150MB WinRM allocates per shell on older versions of
+Windows (prior to Windows Server 2012) -- this can slow down
 bootstrap. Optionally increase the memory limit to 300MB with the following command:
 
     winrm set winrm/config/winrs @{MaxMemoryPerShellMB="300"}
```

Comment:
https://github.com/chef/chef-provisioning-aws/pull/93#discussion_r24371833

Should this be higher than 300mb?